python/ray/plasma/plasma.py

```diff
@@ -23,7 +24,7 @@ def random_name():
 def start_plasma_store(plasma_store_memory=DEFAULT_PLASMA_STORE_MEMORY,
                        use_valgrind=False, use_profiler=False,
                        stdout_file=None, stderr_file=None,
-                       plasma_directory=None, hugepages_enabled=False):
+                       plasma_directory=None, huge_pages_enabled=False):
```

Comment:
I really don't like to break up `hugepages`, because that's how it appears, as a single word, everywhere: under procfs, under sysfs, and under /var/lib, when mounted by hugeadm. I'd really prefer this to be one word. I think users familiar with hugepages will find this break unnatural.